### DISCUSSION TITLE
IP Whitelisting while development is going

### DISCUSSION THREAD

**Discussion by RicPiccolo** [SalesforceCommerceCloud/sfcc-ci#524] [Team Posts]
Hi everyone!
Could you please tell me how to whitelist an IP address to have access without Http-authentification in the case authentication is configured. And also why username:password@https://placeholder.demandware.com does not work (Passing auth with credentials in URL)?

**Comment by zlatinz** [comment]
It does not work, because you do not use proper syntax.

It should follow the pattern: https://gooduser:secretpassword@www.example.com
e.g.
https://gooduser:secretpassword@httpbin.org/basic-auth/gooduser/secretpassword

You cannot whitelist IP addresses to not have authentication, if the site is configured to require it.

**Comment by RicPiccolo** [comment]
It was my mistake in the message but I used the right pattern, as you provided. And it does not work - it asks Password anyway. So question is - should it work?
